addons/website_forum/views/forum.xml

```diff
@@ -92,7 +92,7 @@
                                     </group>
                                 </group>
                                 <group>
-                                    <field name="description" nolabel="1" placeholder="Description visible on website"/>
+                                    <field name="description" nolabel="1" placeholder="Description visible on website" colspan="2"/>
```

Comment:
Note to myself: The proper code here should be removing the group (only one field with `nolabel="1"` inside and the view is not extended to add anything anywhere). As a stable fix, I guess adding colspan="2" is better. I'll make the proper change for the master forward-ported version.